bats/tests/k8s/helm-install-rancher.bats

```diff
@@ -13,12 +13,95 @@ local_setup() {
     helm repo update
 }
 
+# Get the minimum Kubernetes version the given rancher-latest/rancher helm chart
+# does _not_ support; i.e. the chart supports < [output].
+get_chart_unsupported_version() {
+    output="" # Hack: tell shellcheck about this variable
+    local chart_version=$1
+    run helm show chart rancher-latest/rancher --version "$chart_version"
+    assert_success || return
+    run awk '/^kubeVersion:/ { $1 = ""; print }' <<<"$output"
+    assert_success || return
+    # We only support kubeVersion of form "< x.y.z"
+    assert_output --regexp '^[[:space:]]*<[[:space:]]*[^[:space:]]+$' || return
+    awk '{ print $2 }' <<<"$output"
+}
+
+# Try to determine the best chart version for the current Kubernetes version
```

Comment:
I think `best` should be defined here:

The function returns `$RD_RANCHER_IMAGE_TAG` if it is set (and compatible), or otherwise the oldest chart version that supports `$RD_KUBERNETES_PREV_VERSION`.

If the Kubernetes version cannot be supported, then it calls `skip_k3s_version`.